webxr/xrRigidTransform_constructor.https.html

```diff
@@ -65,43 +65,46 @@
   let transform = new XRRigidTransform(
       createDOMPoint([1.0, 2.0, 3.0]),
       createDOMPoint([1.1, 2.1, 3.1, 1.0]));
-  checkTransform(transform);
-  checkDOMPoint(transform.position, 1.0, 2.0, 3.0, 1.0);
-  assert_approx_equals(quaternionLength(transform.orientation), 1.0, FLOAT_EPSILON);
+  checkTransform(transform, "Arbitrary transform");
+  checkDOMPoint(transform.position, 1.0, 2.0, 3.0, 1.0, "Arbitrary transform position");
+  assert_approx_equals(quaternionLength(transform.orientation), 1.0, FLOAT_EPSILON,
+                       "Arbitrary transform is normalized");
 
   // test creating identity transform
   let identity = new XRRigidTransform();
-  checkTransform(identity);
-  checkDOMPoint(identity.position, 0.0, 0.0, 0.0, 1.0);
-  checkDOMPoint(identity.orientation, 0.0, 0.0, 0.0, 1.0);
-
-  // test creating transform with quaternion of length 0
-  // constructor should not crash
-  let zeroLength = new XRRigidTransform(
```

Comment:
cc @alcooper91 this test is incorrect, see https://immersive-web.github.io/webxr/#normalize